openapi/components/schemas/KycDocument/IdentityMatches.yaml

```
@@ -101,6 +101,10 @@ properties:
     description: Specifies if an identity document has matching face proof.
     type: boolean
     example: false
+  isTampered:
+    description: Specifies if an identity document is suspicious of being tampered.
```

nitpick: 'suspicious' doesn't really work here

if we are certain that the document has been tampered with 
```suggestion
    description: Specifies if an identity document has been tampered with.
```
if we are not certain that the document has been tampered with 
```suggestion
    description: Specifies if an identity document may have been tampered with.
```